ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -121,7 +115,7 @@ void requestBannerAd(@NonNull AdSize adSize, @NonNull MediationAdRequest mediati
     } catch (IllegalArgumentException exception) {
       Log.w(TAG, "Failed to load ad from Vungle.", exception);
       if (bannerListener != null) {
-        bannerListener.onAdFailedToLoad(adapter, AdRequest.ERROR_CODE_INVALID_REQUEST);
+        bannerListener.onAdFailedToLoad(adapter.get(), AdRequest.ERROR_CODE_INVALID_REQUEST);
```

Comment:
do you sure that it's safe to pass MediationBannerAdapter as null to bannerListener?
In case MediationBannerAdapter was garbage-collected

Reply:
null check updated.